### PR TITLE
Ignore Horizon routes

### PR DESCRIPTION
**Pull request by adampatterson**
I noticed that the routes for Telescope were ignored and thought that it would also be good to exclude Horizons routes.

### CODE REVIEW

**Comment by zepfietje** [comment]
Thanks, @adampatterson, I agree it would be good to also ignore Horizon routes by default.
However, as the list of routes to be ignored gets longer, maybe we should think of a configurable solution? Any ideas?

**Comment by adampatterson** [comment]
@zepfietje   I like that, you may not want to track dashboard or admin views.

I'll update my PR over the next few days to pull from an array in the pirsch config file.

**Comment by zepfietje** [comment]
Not tracking dashboard or admin views is quite easy already by just removing (or not adding) the middleware from those routes. So this feature would be most useful for third-party package routes. But yes, might as well use it to simplify ignoring a group of routes.

Let me know when you're ready by marking this PR as ready for review. 😊

File: src/Http/Middleware/TrackPageview.php [hunk 1]
```diff
@@ -21,8 +21,10 @@ public function handle(Request $request, Closure $next): mixed
             return $response;
         }
 
-        if (str_starts_with($request->route()->uri, 'telescope/')) {
```
Comment (Autive) [inline src/Http/Middleware/TrackPageview.php:24]: Wouldn't this be a bit cleaner?

```php
		if ( in_array( $request->route()->uri, config( 'pirsch.excluded_routes' ) ) ) {
			return $response;
		}
```

**Comment by zepfietje** [comment]
Is this ready for review, @adampatterson? 🙂

**Comment by adampatterson** [comment]
@zepfietje Iet me double check this weekend.

I had Pirsch running on a server that Oracle decided to delete on me 🙃

**Comment by zepfietje** [comment]
No rush, @adampatterson! Just wondered what the status of this PR was :)

**Comment by zepfietje** [comment]
Replaced by https://github.com/pirsch-analytics/laravel-pirsch/pull/9.

**Comment by adampatterson** [comment]
> Replaced by #9.

Sorry, I no longer had a way of testing with Pirsh.

**Comment by zepfietje** [comment]
No worries, @adampatterson.

**Comment by adampatterson** [comment]
@zepfietje what's the best method for testing Pircsh since the trial is limited to 30 days?

**Comment by zepfietje** [comment]
I'm sure @Kugelschieber can help you test it for a little longer if needed. 🙂

**Comment by Kugelschieber** [comment]
@adampatterson I can extend your trial. Can you send an email to support@pirsch.io with your account email address?

**Comment by adampatterson** [comment]
@zepfietje I went ahead and updated my branch with the latest changes and tested in Pirsh and everything seems to be working as expected but I did notice something odd with the unit tests.

I have never used PEST before so duplicated a few previous tests and set the configs as if I were going to exclude a custom route or header.

https://github.com/adampatterson/laravel-pirsch/blob/middleware/tests/TrackPageviewTest.php#L50

And they passed, so naturally I tried to break the test and it kept passing.

I then went back to a previous test and modified the `excluded_routes` values and the test still passed.

I then modified the test it's self and it kept passing.

```php
return [
    'excluded_routes' => [
        //
    ],
,
```

```php
it('skips Telescope', function () {
    Pirsch::spy();

    Route::middleware(TrackPageview::class)
        ->get('thisshouldcall/test', fn() => 'Hello World');

    $this->get('/thisshouldcall/test');

    Pirsch::shouldNotHaveBeenCalled();
});
```

I would expect this to fail since the route is not excluded.

**Pull request closed without merging**